README.md

```diff
@@ -113,6 +114,68 @@ NinchatSession session = builder.create();
 
 See [Ninchat API Reference](https://github.com/ninchat/ninchat-api/blob/v2/api.md) for information about the API's outbound Actions and inbound Events.
 
+### Open ninchat using existing session
+
+Ninchat support session resuming. In order to support session resuming feature in your application, you need to create and pass ( *optional* )  `NinchatSessionCredentials` object instance in the builder script
+
+```java
+SharedPreferences pref = getApplicationContext().getSharedPreferences("pref", MODE_PRIVATE);
+
+NinchatSessionCredentials sessionCredentials = new NinchatSessionCredentials(
+  pref.getString("user_id", null),
+  pref.getString("user_auth", null), 
+  pref.getString("session_id", null));
+
+NinchatSession.Builder builder = new NinchatSession.Builder(
+  getApplicationContext(),
+  getString(R.string.ninchat_configuration_key))
+                .setConfiguration(this.ninchatConfiguration)
+							  // ........................................
+						    // ..if we want session to be persistence..
```

Comment:
I believe it should be "If we want session to be persistent" or "If session should be established using saved credentials"

Reply:
Maybe a single comment would be enough (without extra dotted comment lines)